relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics_mobile.snap

```diff
@@ -74,7 +74,44 @@ expression: "(&event.value().unwrap().spans, metrics)"
             tags: ~,
             origin: ~,
             profile_id: ~,
-            data: ~,
+            data: SpanData {
+                app_start_type: ~,
+                browser_name: ~,
+                code_filepath: ~,
+                code_lineno: ~,
+                code_function: ~,
+                code_namespace: ~,
+                db_operation: ~,
+                db_system: ~,
+                environment: ~,
+                release: ~,
+                http_decoded_response_content_length: ~,
+                http_request_method: ~,
+                http_response_content_length: ~,
+                http_response_transfer_size: ~,
+                resource_render_blocking_status: ~,
+                server_address: ~,
+                cache_hit: ~,
+                cache_item_size: ~,
+                http_response_status_code: ~,
+                ai_input_messages: ~,
+                ai_completion_tokens_used: ~,
+                ai_prompt_tokens_used: ~,
+                ai_total_tokens_used: ~,
+                ai_responses: ~,
+                thread_name: ~,
+                segment_name: ~,
+                ui_component_name: ~,
+                url_scheme: ~,
+                user: ~,
+                replay_id: ~,
+                sdk_name: ~,
+                other: {
+                    "frames.delay": I64(
+                        123,
+                    ),
+                },
```

Comment:
Am I doing something incorrect to have this field appear in the `other` property compared to the other keys?